### ISSUE TITLE
Torchvision backend does not work

### ISSUE THREAD

**Issue by alpargun** [facebookresearch/SlowFast#579]
I am not able to run torchvision as the backend while running SlowFast. I came across some people mentioning that they managed to run torchvision after they reinstalled it from source, however, I am still not able to use it after reinstalling it from the source.

You can find the packages I have installed below and I would really appreciate it if you can share your versions in case you are able to run SlowFast with torchvision as the backend.

Here is my _pip list_ output:

Package                 Version   Location
----------------------- --------- ---------------------------------
absl-py                 1.1.0
antlr4-python3-runtime  4.9.3
av                      9.2.0
black                   22.3.0
brotlipy                0.7.0
cachetools              5.2.0
certifi                 2022.6.15
cffi                    1.14.6
charset-normalizer      2.1.0
click                   8.1.3
cloudpickle             2.1.0
colorama                0.4.5
cryptography            37.0.2
cycler                  0.11.0
Cython                  0.29.30
decorator               5.1.1
detectron2              0.6       /home/alp/Desktop/detectron2_repo
fairscale               0.4.6
fonttools               4.34.4
future                  0.18.2
fvcore                  0.1.5
google-auth             2.9.0
google-auth-oauthlib    0.4.6
grpcio                  1.47.0
hydra-core              1.2.0
idna                    3.3
imageio                 2.9.0
imageio-ffmpeg          0.4.7
importlib-metadata      4.12.0
iopath                  0.1.9
joblib                  1.1.0
kiwisolver              1.4.3
Markdown                3.3.7
matplotlib              3.5.2
mkl-fft                 1.3.1
mkl-random              1.2.2
mkl-service             2.4.0
moviepy                 1.0.3
mypy-extensions         0.4.3
networkx                2.8.4
numpy                   1.22.3
oauthlib                3.2.0
olefile                 0.46
omegaconf               2.2.2
opencv-python           4.6.0.66
packaging               21.3
pandas                  1.4.3
parameterized           0.8.1
pathspec                0.9.0
Pillow                  8.2.0
pip                     21.2.4
platformdirs            2.5.2
portalocker             2.4.0
proglog                 0.1.9
protobuf                3.19.4
psutil                  5.9.1
pyasn1                  0.4.8
pyasn1-modules          0.2.8
pycocotools             2.0.4
pycparser               2.21
pydot                   1.4.2
pyOpenSSL               22.0.0
pyparsing               3.0.9
PySocks                 1.7.1
python-dateutil         2.8.2
pytorchvideo            0.1.5
pytz                    2022.1
PyYAML                  6.0
requests                2.28.1
requests-oauthlib       1.3.1
rsa                     4.8
scikit-learn            1.1.1
scipy                   1.9.0rc2
setuptools              61.2.0
simplejson              3.17.6
six                     1.16.0
sklearn                 0.0
slowfast                1.0       /home/alp/Desktop/SlowFast
tabulate                0.8.10
tensorboard             2.9.1
tensorboard-data-server 0.6.1
tensorboard-plugin-wit  1.8.1
termcolor               1.1.0
threadpoolctl           3.1.0
timm                    0.5.4
tomli                   2.0.1
torch                   1.12.0
torchaudio              0.12.0
torchvision             0.13.0
tqdm                    4.64.0
typing_extensions       4.3.0
urllib3                 1.26.10
Werkzeug                2.1.2
wheel                   0.37.1
yacs                    0.1.8
zipp                    3.8.0
